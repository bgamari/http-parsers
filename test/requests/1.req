GET /post/love-supreme-comes-alive-unearthed-photos HTTP/1.0
Host: kuvo.org
Accept-Encoding: x-gzip, gzip, deflate
User-Agent: CCBot/2.0 (http://commoncrawl.org/faq/)
Accept-Language: en-us,en-gb,en;q=0.7,*;q=0.3
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8

