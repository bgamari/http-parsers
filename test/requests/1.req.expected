Right
  Request
    { reqMethod = GET
    , reqUri = "/post/love-supreme-comes-alive-unearthed-photos"
    , reqVersion = HTTP / 1.0
    , reqHeaders =
        [ ( "Host" , "kuvo.org" )
        , ( "Accept-Encoding" , "x-gzip, gzip, deflate" )
        , ( "User-Agent" , "CCBot/2.0 (http://commoncrawl.org/faq/)" )
        , ( "Accept-Language" , "en-us,en-gb,en;q=0.7,*;q=0.3" )
        , ( "Accept"
          , "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8"
          )
        ]
    }